src/tensors/cpu/tensor_operators.cpp

```diff
@@ -275,11 +276,8 @@ void TransposeFirst3In4(Tensor out, Tensor in, const std::vector<int>& vAxis) {
       }
     }
   }
-#else
```

Comment:
Is there a reason to remove this? It seems unrelated to clang?

Reply:
There's exactly one call to that function in the Marian code, and that's farther down in this file. That call is already guarded by `#if MKL_FOUND`. Should anyone ever decide to call the function from somewhere else without using MKL, compilation should fail, not running the program at some random point. This change is related to clang, because otherwise we get an unused variable warning.